### PR TITLE
Support TYPO3 v10 LTS

### PR DESCRIPTION
**Pull request by lukasnetz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@ stages:
   script:
     - php .Build/bin/phpunit -c ./phpunit.xml --coverage-clover=coverage.xml
     - bash <(curl -s https://codecov.io/bash)
+  before_script:
+    - if [ "$dependencies" = "lowest" ]; then composer update --prefer-lowest --prefer-stable -n; fi;
```
Comment (markuspoerschke) [inline .travis.yml]: Can we move this command to `install`? I think this will improve the reading flow.

Comment (lukasnetz) [reply .travis.yml]: Moved it to install in commit https://github.com/cvc-digital/typo3-webauthn/pull/8/commits/ad1c8c436c3c2c3ca8cc5596cd4cbbef3a62e03e

**Review by markuspoerschke — APPROVED** [review]

File: Classes/Controller/BackendController.php [hunk 2]
```diff
@@ -47,7 +49,10 @@ class BackendController extends ActionController
 
     public function __construct(KeyRepository $keyRepository, BackendUserRepository $backendUserRepository)
     {
-        parent::__construct();
+        if (Comparator::greaterThanOrEqualTo(Versions::getVersion('typo3/cms-core'), '10')) {
```
Comment (marclindemann) [inline Classes/Controller/BackendController.php:52]: why we need  extra dependencies here? 
`  if (version_compare(TYPO3_version, '10.0.0', '>=')) {
`
this does the same job

Comment (markuspoerschke) [reply Classes/Controller/BackendController.php:52]: We are using the `Comparator` and `Versions` class also in the `Factory` class: https://github.com/cvc-digital/typo3-webauthn/pull/8/files#diff-730b305699c79dc320b26ffe41591fcaR53-R56

The reason, why we need `Versions` is that `web-auth/webauthn-lib` does not provide any version information.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.2.0
+
+### Added
+
+* Support TYPO3 v10.3
```
Comment (marclindemann) [inline CHANGELOG.md]: v.10.4 @markuspoerschke

Comment (markuspoerschke) [reply CHANGELOG.md]:
```suggestion
* Support TYPO3 v10.4
```

**Review by markuspoerschke — APPROVED** [review]

**Review by marclindemann — APPROVED** [review]

**Review by robinkitzelmann — APPROVED** [review]

**Pull request merged**